### DISCUSSION TITLE
StreetComplete for Android - Feature request.

### DISCUSSION THREAD

**Discussion by xuid0** [streetcomplete/StreetComplete#6063] [Ideas]
![Screenshot_20241230-105342](https://github.com/user-attachments/assets/f977580b-f1e2-4ebc-8528-6ec001e53c6a)

(Attached screenshot of GitHub app for Android doing the feature I'm discussing for help understanding).

I notice the app does not have a app lock for when it's opened could we start the discussion about enabling or adding in the ability to lock access down to the app until a Biometric is used to unlock/access?

Otherwise the app runs really well on my device. If I ever win lots of money I'll definitely be sending some coin to the developers of StreetComplete a bit of money. Thank you keep doing what your doing, even your GitHub is a professional experience.

**Comment by mnalis** [comment]
> I notice the app does not have a app lock for when it's opened could we start the discussion about enabling or adding in the ability to lock access down to the app until a Biometric is used to unlock/access?

Why would you need to lock it? All data you map is going to be uploaded publicly, thus available for whole world to see (and not just persons having access to your phone).

Also what is that "app lock" of yours? My android does not seem to have that. (There are many apps which provide ability to hide applications behind some access controls, some shipped by some vendors, some available by third parties)

**Reply by xuid0** [reply]
> > I notice the app does not have a app lock for when it's opened could we start the discussion about enabling or adding in the ability to lock access down to the app until a Biometric is used to unlock/access?
> 
> Why would you need to lock it? All data you map is going to be uploaded publicly, thus available for whole world to see (and not just persons having access to your phone).
> 
> Also what is that "app lock" of yours? My android does not seem to have that. (There are many apps which provide ability to hide applications behind some access controls, some shipped by some vendors, some available by third parties)

I don't want to elaborate on my reasoning I was only making a suggestion for a feature which would be nice to have. I'm guessing it's not on the roadmap thanks for replying.

**Reply by mnalis** [reply]
Sorry if there was misunderstanding, I was just trying to gather more data in order to understand the use case and with what exactly is the issue you're having.
(Of course you're under no obligation to provide any data if you don't want to)

**Reply by xuid0** [reply]
Apologies I do not wish to provide any further discussion as I totally understand your point of view for not wanting to create more work etc.

It was intended as a friendly suggestion on improving the app overall it was not an attack on the Developers or anything like that. And i don't want to provide context for the use case. Thanks for understanding I will close this issue now.
